apps/src/templates/studioHomepages/Sections.jsx

```diff
@@ -58,23 +75,33 @@ const Sections = React.createClass({
             result={this.state.sectionsResult}
             nameOrId={this.state.sectionsResultName}
           />
-          {sections.length > 0 && (
+          {isTeacher && numTeacherSections > 0 && sectionFlow2017 && (
+            <SectionsPage
+              className="sectionPage"
+              teacherHomepage={this.props.teacherHomepage}
+            />
+          )}
```

Comment:
Should this component be called something other than SectionsPage at this point?